tests/graphical.rs

```diff
@@ -803,18 +815,46 @@ fn related() -> Result<(), MietteError> {
  3 │     here
    ╰────
   help: try doing it better next time?
-
-Error: oops::my::bad
-
-  × oops!
-   ╭─[bad_file.rs:1:1]
- 1 │ source
-   · ───┬──
-   ·    ╰── this bit here
- 2 │   text
-   ╰────
-  help: try doing it better next time?
-
+╭─There were 2 related diagnostics:
+├─ 1.Error: oops::my::bad
+│ 
+│ 
```

Comment:
Is doubling the number of empty new lines deliberate or an oversight?
It does hurt the sense of visual grouping for me.